pages/public/rush.tsx

```diff
@@ -10,5 +13,80 @@ export const getStaticProps: GetStaticProps = async () => {
 };
 
 export default function Page() {
-  return <div>Rush</div>;
+  return (
+    <div>
+      <div className="w-full h-96 flex flex-col justify-center items-center gap-6 bg-[#3E395F] text-white">
```

Comment:
don't use bracketed colors, we want tailwind default colors for simplicity, and at another point we can consider color change